File: C/Colleen.c
#include <stdio.h>
// ceci est un commentaire
void fct(){
	char *a = "#include <stdio.h>%1$c// ceci est un commentaire%1$cvoid fct(){%1$c%2$cchar *a = %3$c%4$s%3$c;printf(a, 10, 9, 34, a);%1$c}%1$cint main(){%1$c%2$c//le deuxieme commentaire%1$c%2$cfct();%1$c}%1$c";printf(a, 10, 9, 34, a);
}
int main(){
	//le deuxieme commentaire
	fct();
}

File: PYTHON/Colleen.py
#premier commentaire
def fct(a):
    print(a.format('    ', 10, 34, a))
if __name__ == '__main__':
    #commentaire interne
    a = "#premier commentaire{1:c}def fct(a):{1:c}{0}print(a.format('    ', 10, 34, a)){1:c}if __name__ == '__main__':{1:c}{0}#commentaire interne{1:c}{0}a = {2:c}{3}{2:c}{1:c}{0}fct(a)"
    fct(a)
